### ISSUE TITLE
cf.set_config_file(offline=True) adds 12mb of javascript to notebook

### ISSUE THREAD

**Issue by manycoding** [santosjorge/cufflinks#173]
In any notebook, these lines increase the size of notebook on 12mb:
```
import cufflinks
cufflinks.set_config_file(offline=True)
```
cufflinks 0.14.6
plotly 3.7.1

To reproduce, I run `pipenv install cufflinks jupyterlab` and put those lines in a notebook.

Maybe connected with https://github.com/plotly/plotly.py/issues/969

**Comment by manycoding** [comment]
[Untitled.ipynb.zip](https://github.com/santosjorge/cufflinks/files/3024065/Untitled.ipynb.zip)

**Comment by manycoding** [comment]
```
-i https://pypi.org/simple
appnope==0.1.0 ; sys_platform == 'darwin'
attrs==19.1.0
backcall==0.1.0
bleach==3.1.0
certifi==2019.3.9
chardet==3.0.4
colorlover==0.3.0
cufflinks==0.14.6
decorator==4.4.0
defusedxml==0.5.0
entrypoints==0.3
idna==2.8
ipykernel==5.1.0
ipython-genutils==0.2.0
ipython==7.4.0 ; python_version >= '3.3'
ipywidgets==7.4.2
jedi==0.13.3
jinja2==2.10
jsonschema==3.0.1
jupyter-client==5.2.4
jupyter-core==4.4.0
jupyterlab-server==0.2.0
jupyterlab==0.35.4
markupsafe==1.1.1
mistune==0.8.4
nbconvert==5.4.1
nbformat==4.4.0
nose==1.3.7
notebook==5.7.7
numpy==1.16.2
pandas==0.24.2
pandocfilters==1.4.2
parso==0.3.4
pexpect==4.6.0 ; sys_platform != 'win32'
pickleshare==0.7.5
plotly==3.7.1
prometheus-client==0.6.0
prompt-toolkit==2.0.9
ptyprocess==0.6.0 ; os_name != 'nt'
pygments==2.3.1
pyrsistent==0.14.11
python-dateutil==2.8.0
pytz==2018.9
pyzmq==18.0.1
requests==2.21.0
retrying==1.3.3
send2trash==1.5.0
six==1.12.0
terminado==0.8.2
testpath==0.4.2
tornado==6.0.2
traitlets==4.3.2
urllib3==1.24.1
wcwidth==0.1.7
webencodings==0.5.1
widgetsnbextension==3.4.2
```

**Comment by santosjorge** [comment]
This was a great find. I can't believe no one complained about it before. Thank you @manycoding

**Comment by eddy-geek** [comment]
For anyone hitting this, the fix is tagged 0.15.

Reduced my ipynb's by 5.7MB -- I was simply using `cufflinks.go_offline()` which now default to calling  `plotly.offline.init_notebook_mode(connected=True)`
